library/src/scripts/headers/titleBarStyles.ts

```diff
@@ -92,6 +92,10 @@ export const titleBarVariables = useThemeCache(() => {
         },
     });
 
+    const navAlignment = makeThemeVars("navAlignment", {
+        alignment: "left",
```

Comment:
```suggestion
        alignment: "left" as "left" | "center",
```

Better to tighten up this string a bit.